geonode/layers/models.py

```diff
@@ -339,8 +339,8 @@ def attribute_config(self):
         if (visible_attributes.count() > 0):
             cfg["getFeatureInfo"] = {
                 "fields": [lyr.attribute for lyr in visible_attributes],
-                "propertyNames": dict([(lyr.attribute, lyr.attribute_label) for lyr in visible_attributes]),
-                "displayTypes": dict([(lyr.attribute, lyr.featureinfo_type) for lyr in visible_attributes])
+                "propertyNames": dict((lyr.attribute, lyr.attribute_label) for lyr in visible_attributes),
+                "displayTypes": dict((lyr.attribute, lyr.featureinfo_type) for lyr in visible_attributes)
```

Comment:
The same change as before can be applied to all `dict(...)` definitions like this as in my last review. Please check for all occurrences of this in order to simplify.